catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/resource/download/ReliableResourceDownloaderTest.java

```diff
@@ -61,6 +65,8 @@
 import ddf.catalog.resource.ResourceNotSupportedException;
 import ddf.catalog.resourceretriever.ResourceRetriever;
 
+@RunWith(PowerMockRunner.class)
```

Comment:
Can you double check the JaCoCo coverage report here: `target/site/project-info/jacoco/ddf.catalog.resource.download/ReliableResourceDownloader.html`? It looks like those lines aren't covered for some reason.